superset/models/dashboard.py

```diff
@@ -334,6 +323,29 @@ def alter_positions(dashboard, old_to_new_slc_id_dict):
                     old_slc_id_str
                 ]
 
+        # since PR #9109, filter_immune_slices and filter_immune_slice_fields
```

Comment:
Is there any way we can avoid having this special logic? I'm worried about code in Superset getting (even more) bloated. When will `create_from_import` have the extra fields? Will it always? Is it ok to just not support this case and warn that your change is not backwards compatible?

Reply:
`export dashboard` and `import dashboard` is a widely used feature in Superset. it helps ppl to move dashboard from different data system (and both of them use Superset). If someone wants to copy a dashboard from an older system which didn't upgrade recently, but the other system used updated version, this PR can make their export - import routine keep working.

if the backward compatible only introduce a couple of lines, I would like to keep it. But if it will introduce a lot of overhead, i will not maintain it.